openlibrary/catalog/get_ia.py

```diff
@@ -1,14 +1,16 @@
 from openlibrary.catalog.marc.marc_binary import MarcBinary
 from openlibrary.catalog.marc.marc_xml import MarcXml
 from openlibrary.catalog.marc import parse
+from infogami import config, load_config
 from lxml import etree
 import xml.parsers.expat
 import urllib2, os.path, socket
 from time import sleep
 import traceback
 from openlibrary.core import ia
 
-IA_BASE_URL = 'https://archive.org'
+load_config('conf/openlibrary.yml')
```

Comment:
@mekarpeles I'm not completely happy about this line, `openlibrary/plugins/importapi/code.py` already has config available via `config.get()`,  but `openlibrary/catalog/get_ia.py` does not :man_shrugging:   I don't understand the purpose of https://github.com/internetarchive/openlibrary/blob/master/openlibrary/config.py either, it looks like a confusingly named / name collision version of `infogami.config` .  This seems to work, but I'm not sure I fully understand how config is meant to be shared, or passed around parts of OL.